### ISSUE TITLE
default testing code for standard types in python3 / R

### ISSUE THREAD

**Issue by trevorcampbell** [UBC-DSCI/nbgrader#12]
Should create a default suite of test templates for Python3 and R to be shipped with nbgrader

**Comment by wiwang** [comment]
@trevorcampbell 
the templates you mentioned here is the `tests.yml` which `nbgrader` use to fill and generate the test code?

**Comment by wiwang** [comment]
@trevorcampbell  could you please the question above?  during the autogenerating Yiki's worksheets, I always need to copy this test.yml to the source folder. thank you

**Comment by trevorcampbell** [comment]
@wiwang yes, you'll need to have a separate copy of `tests.yml` for each `source/worksheet_*` and `source/tutorial_*` folder. I made it this way because instructors may want to use different tests for each worksheet.

My original post here:

> Should create a default suite of test templates for Python3 and R to be shipped with nbgrader

was to make sure that we create a *default* `tests.yml` file that contains pre-made tests for common types (float, int, string, list, dict, set, ...... anything else you can think of)
